docs/program/apis/sessions.md

```diff
@@ -0,0 +1,86 @@
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session", "sessions", "session management"]
+---
+
+The session management API provides support for robot session management.
+
+A session:
+
+- allows a client to express that it is actively connected or authenticated to `viam-server` on your robot
+- supports stopping moving components when the connection is no longer active.
+
+### Purpose of session management
+
+When controlling a robot or fleet with Viam, you want a way to understand the presence of the clients that are communicating with and authenticated to each robot's `viam-server` through Viam's various [resource APIs](/program/apis/).
+
+The presence of these clients at one point in time is called a *session*.
+A session technically is any presence mechanism at the RDK application layer maintained by any client with `viam-server`.
+
+Session management allows for safer operation of robots that physically move.
+Specifically, without session management, controls that would be "sticky" (for example [`SetPower()` of a base](/components/base/#setpower)) based on the last input of a **client**, meaning the API request from the client sets the flow of electricity on a part of the robot and then does not time out, could lead to **having a robot trying to continue doing what it was told to do forever.**
+
+You do not want that to happen.
+
+#### Clients and Viam's client SDKs
+
+*Client* has multiple meanings for a Viam robot.
+Essentially, it's anything that is receiving the information served by `viam-server` running on the robot.
+
+A **client** can be an SDK script controlling the robot, an input controller, or just the different resources on the robot talking amongst themselves.
+For example, if you use Viam's module registry to [add modular resources to your robot](/extend/modular-resources/), the clients of your robot in its lifetime will include the "model servers" you instantiate on your robot for individual resources.
+
+Viam's session management API is your built-in solution to this.
+Your client maintains the session, telling the `viam-server` instance that it is still present every so often, or staying within the heartbeat window.
+
+Your client must send at least one session heartbeat within this window.
+As soon as the window lapses/expires, the server will safely stop all resources that are marked for safety monitoring that have been last used by that session, and no others; a lapsed client will attempt to establish a new session immediately prior to the next operation it performs.
+
+## Usage
+
+Usage of the session management API differs across [Viam's SDKS](/program/).
+
+### Manage on-robot sessions
+
+{{< tabs >}}
+{{% tab name="Go" %}}
+
+To use the [Session Management API](https://pkg.go.dev/go.viam.com/rdk/session) the Go Client SDK provides, use your [`RobotClient`](/program/apis/#robot-api) instance (client of the Robot API) to instantiate a `SessionsManager` with `SessionManager()`.
```

Comment:
No need to instantiate a separate `SessionsManager` client - one will be instantiated when you start a robot client. Instead, you should disable sessions by passing the following option to the robot client:

```go
robot, err := client.New(ctx, "my-robot-address", logger, client.WithDisableSessions(), ...)
```

Reply:
done/changed